addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1827,7 +1857,7 @@ class SnippetsMenu extends Component {
 
     static template = "web_editor.SnippetsMenu";
 
-    static components = { Toolbar, LinkTools };
+    static components = { Toolbar, LinkTools, AddSnippetDialog };
```

Comment:
I think there is no need to add `AddSnippetDialog` to the components. It's not called directly in the `SnippetsMenu` template, it's only called through the dialog service.